pkg/translator/opencensus/go.mod

```diff
@@ -0,0 +1,15 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/opencensus
+
+go 1.16
+
+require (
+	github.com/census-instrumentation/opencensus-proto v0.3.0
+	github.com/golang/protobuf v1.5.2
+	github.com/google/go-cmp v0.5.6
+	github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal v0.0.0-20210820201140-13cd75901f9f
```

Comment:
Here overwrite with `	github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal v0.0.0-00010101000000-000000000000
`

then at the bottom of the file add:

replace github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal => ../../internal/coreinternal

Reply:
Done